deploy/deploy-systemd.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+BRANCH=$1
+name=
+
+if [ "x$BRANCH" = "x" ]; then
+  echo "Please pass branch to deploy as first argument"
+  exit 1
+fi
+
+if [ "x$BRANCH" = "xproduction" ]; then
+  name=foodsaving-world
+else
+  name=foodsaving-world-dev
+fi
+
+function restart-workers() {
+  sudo systemctl restart "${name}-worker.target"
+}
+
+backend_dir="/var/www/${name}/www"
```

Comment:
Do you intend to change the backend dir from `/home/deploy/` to `/var/www/`? If yes, why?

You would need to update the cronjobs as well: 

![image](https://user-images.githubusercontent.com/4410802/29180504-54447cea-7df8-11e7-9a32-6f4b37033d2e.png)